R-package/tests/testthat/test_parameters.R

```diff
@@ -101,14 +101,83 @@ test_that("training should warn if you use 'dart' boosting, specified with 'boos
         , num_threads = .LGB_MAX_THREADS
     )
     params[[boosting_param]] <- "dart"
+
+    # expect warning
+    # if early_stopping_rounds is specified
```

Comment:
```suggestion
    # warning: early stopping requested
```

Please consolidate this to 1 line.